logstash-core/lib/logstash/environment.rb

```diff
@@ -123,6 +124,7 @@ module Environment
   SETTINGS.on_post_process do |settings|
     # Configure Logstash logging facility. This needs to be done as early as possible to
     # make sure the logger has the correct settings tnd the log level is correctly defined.
+    java.lang.System.setProperty("ls.log.json.strict", settings.get("log.format.json_strict").to_s)
```

Comment:
I think we should be consistent here - looks like all the other System properties are `ls.#{property_name}`, so I think `ls.log.format.json_strict` might be easier to follow

Reply:
Fixed by 923756094db5c8c89bbfe2115567cf98606b94dd